paris/src/testDebug/java/com/airbnb/paris/proxies/TextViewStyleApplierTest.kt

```diff
@@ -54,6 +59,18 @@ class TextViewStyleApplierTest {
         assertFalse(view.compoundDrawables[0].bounds.isEmpty)
     }
 
+    @Test
+    fun fontFamily_string() {
+        applier.apply(R.style.Test_TextViewStyleApplier_FontFamily_String)
+        assertTypefaceEquals(Typeface.create("sans-serif-bold", Typeface.NORMAL), view.typeface)
+    }
+
+    @Test
+    fun fontFamily_fontReference() {
+        applier.apply(R.style.Test_TextViewStyleApplier_FontFamily_Resource)
+        assertTypefaceEquals(ResourcesCompat.getFont(context, R.font.roboto_regular), view.typeface)
```

Comment:
Isn't `roboto_regular` the default as well?

Reply:
Not really - this refers to `roboto_regular.ttf` which I placed in `paris/src/debug/res/font/`